src/blueprints/Do.Blueprints.Service.Application/Greeting/Swagger/SwaggerGreetingFeature.cs

```diff
@@ -16,5 +17,26 @@ public void Configure(LayerConfigurator configurator)
                 return Task.CompletedTask;
             });
         });
+
+        configurator.ConfigureSwaggerGenOptions(swaggerGenOptions =>
+        {
+            swaggerGenOptions.CustomSchemaIds(t =>
+            {
+                string[] splitedNamespace = t.Namespace!.Split(".");
+                string name = t.Name;
+
+                if (t.IsNested)
+                {
+                    name = t.FullName?
+                        .Replace($"{t.Namespace}.", "")
+                        .Replace("Controller", "")
+                        .Replace("+", ".")!;
+                }
+
+                return splitedNamespace.Length > 1
+                    ? $"{string.Join('.', splitedNamespace.Skip(1))}.{name}"
+                    : name;
+            });
+        });
```

Comment:
greeting is not correct feature to configure schema ids, we should have a better fit for this one. please discuss it with @mouseless/developers